tools/releaseBuild/Images/microsoft_powershell_windowsservercore/PowerShellPackage.ps1

```diff
@@ -12,13 +12,16 @@ param (
     [string] $destination = "$env:WORKSPACE",
 
     [ValidateSet("win7-x64", "win7-x86", "win-arm", "win-arm64", "fxdependent", "fxdependent-win-desktop")]
-    [string]$Runtime = 'win7-x64',
+    [string] $Runtime = 'win7-x64',
+
+    [ValidateSet("min-size")]
+    [string] $Configuration = '',
```

Comment:
This parameter has an empty default and a singleton `ValidateSet` attribute. Is this a `[switch]` in disguise?

Reply:
It's possible we will build for other special configurations in future, so better to have a string parameter.

Reply:
On the second thought, it's better to use a switch parameter for the simplicity. We can revisit this if there is another special build/packaging in future.